tests/monotouch-test/ImageIO/CGImageAnimationTest.cs

```diff
@@ -0,0 +1,157 @@
+﻿//
+// Unit tests for CGImageAnimation
+//
+// Authors:
+//	Whitney Schmidt <whschm@microsoft.com>
+//
+// Copyright 2020 Microsoft Corp. All rights reserved.
+//
+using System;
+using System.IO;
+#if XAMCORE_2_0
+using Foundation;
+using System.Threading.Tasks;
+#if MONOMAC
+using AppKit;
+#else
+using UIKit;
+#endif
+using CoreGraphics;
+using ImageIO;
+#else
+using MonoTouch.CoreGraphics;
+using MonoTouch.Foundation;
+using MonoTouch.ImageIO;
+using MonoTouch.UIKit;
+#endif
+using NUnit.Framework;
+
+namespace MonoTouchFixtures.ImageIO {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class CGImageAnimationTest {
+
+		NSUrl imageUrl;
+		NSData imageData;
+		int testValue;
+		CGImageAnimation imageAnimation;
+
+		void MyBlockSetValueZero (nint index, CGImage image, out bool stop)
+		{
+			stop = true;
+			testValue = 0;
+		}
+
+		void MyBlockSetValueOne (nint index, CGImage image, out bool stop)
+		{
+			stop = true;
+			testValue = 1;
+		}
+
+		[TestFixtureSetUp]
+		public void Init ()
+		{
+			TestRuntime.AssertXcodeVersion (11, 4);
+
+			imageUrl = NSBundle.MainBundle.GetUrlForResource ("hack", "gif"); // why not just initialize in declaration?
+			imageData = NSData.FromUrl (imageUrl);
+			imageAnimation = new CGImageAnimation ();
```

Comment:
there's nothing in the file to exclude tvOS or watchOS so it means

* the API is available everywhere - but it's missing the attributes (making available in watchOS 2.0 and tvOS 9.0, which is unlikely to be correct)

* the tests are not executed

Reply:
@spouliot - I've added Watch and TV attributes to `CGImageAnimation.cs` to prevent the API from being included in the min versions of these platforms. Let me know if I understood your comment correctly :)